app/views/practical_supports/_entries.html.erb

```diff
@@ -1,19 +1,23 @@
 <% if patient.practical_supports.present? %>
-  <div class="row practical-support-headers">
-    <div class="col-12">
-      <div class="row">
-        <h5 class="col-3"><%= t 'patient.practical_support.support_needed' %></h5>
-        <h5 class="col-5"><%= t 'patient.practical_support.point_of_contact' %></h5>
-        <h5 class="col-4">Amount</h5>
+  <hr />
+  <% patient.practical_supports.each do |entry| %>
+    <div class="row info-form practical-support-item" id="practical-support-item-<%= entry.id %>">
+      <div class="col practical-support-display-text">
+        <%= practical_support_display_text entry %>
       </div>
-    </div>
-    <div class="col"><!-- Where button would go --></div>
-  </div>
-  <hr/>
 
-  <% patient.practical_supports.each do |entry| %>
-    <%= render 'practical_supports/edit',
-              patient: patient,
-              practical_support: entry %>
+      <div class="col-1 practical-support-note-preview">
+        <%#= entry.most_recent_note_display_text %>
+      </div>
```

Comment:
Is this setup for a future ticket?

Reply:
yeah, next up is pretty much 'put notes in here'.